public_html/wp-content/themes/pattern-directory/src/components/pattern-preview-actions/copy-guide.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Animate, Guide } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+
+const CopyPasteImage = () => (
+	// Wrap the image to avoid the UI shift after the GIF loads
+	<div style={ { height: '220px' } }>
+		<img
+			src="https://shaunandrews.com/wp-content/uploads/2021/03/pattern-paste-demo-2.gif"
```

Comment:
How do we want to ship images? Should they be external to the project somehow?

Reply:
We can include images like this into the theme, I believe they're proxied into the CDN on production.

Reply:
... added an `images` folder in the theme